packages/manager/modules/pci/src/projects/new/components/voucher/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,13 @@
+{
+  "pci_projects_new_voucher_question": "Bon d'achat ?",
+  "pci_projects_new_voucher_form_field": "Entrer un bon d'achat",
+  "pci_projects_new_voucher_form_field_amount": "Montant crédité : {{ amount }}",
+  "pci_projects_new_voucher_form_field_error_invalid": "Code invalide.",
+  "pci_projects_new_voucher_form_field_error_no_longer_valid": "Code invalide.",
```

Comment:
As you have what it seems two different errors, maybe it could be nice for customer to have a distinction between those?